.gitignore

```diff
@@ -43,3 +43,4 @@ pybind11Targets.cmake
 /pybind11/share/*
 /docs/_build/*
 .ipynb_checkpoints/
+tests/main.cpp
```

Comment:
Is this intentional?

Reply:
Yes. That's a temporary file generated by our build system. It was cluttering git status checks.